spec/spec_helper.rb

```diff
@@ -93,6 +93,9 @@
   # Ensure we start with consistent config settings
   config.before(:each) { Spree::Config.products_require_tax_category = false }
 
+  # Tests assume AUD is the default currency.
+  config.before(:each) {Spree::Config.currency = "AUD" }
```

Comment:
`before(:all)` could do the job.

Unfortunately, somehow inside current `spree_core` factories `Spree::Calculator::FlatRate` still take currency from `CURRENCY` variable from `application.yml` and this is not the only failing tests which return `no_shipping_methods_available` error because of currencies mismatch.

Examples of failure:
* `rspec ./spec/jobs/finalize_account_invoices_spec.rb:198`
* ` rspec ./spec/features/consumer/shopping/variant_overrides_spec.rb:152`